sharding-core/src/main/java/io/shardingsphere/core/keygen/DefaultKeyGenerator.java

```diff
@@ -86,23 +88,24 @@
     
     private long lastMilliseconds;
     
-    /**
-     * Set work process id.
-     * 
-     * @param workerId work process id
-     */
-    public static void setWorkerId(final long workerId) {
-        Preconditions.checkArgument(workerId >= 0L && workerId < WORKER_ID_MAX_VALUE);
-        DefaultKeyGenerator.workerId = workerId;
+    @Override
+    public void setKeyGeneratorProperties(final Properties properties) {
+        props = new ShardingProperties(properties);
     }
     
-    /**
-     * Set max tolerate time difference milliseconds.
-     *
-     * @param maxTolerateTimeDifferenceMilliseconds max tolerate time difference milliseconds
-     */
-    public static void setMaxTolerateTimeDifferenceMilliseconds(final int maxTolerateTimeDifferenceMilliseconds) {
-        DefaultKeyGenerator.maxTolerateTimeDifferenceMilliseconds = maxTolerateTimeDifferenceMilliseconds;
+    @Override
+    public Properties getKeyGeneratorProperties() {
```

Comment:
Maybe we don't need `getKeyGeneratorProperties` method, `setProperties` is enough